cli/cmd/generate.go

```diff
@@ -208,7 +210,7 @@ This command can also be run in noninteractive mode. See help output for more de
 			// Load any cached inputs if interactive
 			if cli.InteractiveMode() {
 				cachedOptions := &aws.GenerateAwsTfConfigurationArgs{}
-				iacParamsExpired := cli.ReadCachedAsset(CachedAssetIacParams, &cachedOptions)
+				iacParamsExpired := cli.ReadCachedAsset(CachedAwsAssetIacParams, &cachedOptions)
```

Comment:
@rmoles I'm wondering if we need to re-work this cache loading to have cloud-specific helpers?  Given that we'll be adding the same for Azure/GCP this will likely become a lot of code in this block.

Reply:
Just a note, I think the caching change can be done in another PR. But if @rmoles and @ipcrm can create a ticket description for that.

Reply:
Will certainly target that for a separate issue/PR after we discuss tomorrow.

Reply:
great convo 👍🏽